core/drivers/fabric-driver/.env.docker.template

```diff
@@ -1,8 +1,13 @@
 CONNECTION_PROFILE=<path_to_connection_profile>
 DRIVER_CONFIG=<path_to_config_json>
 RELAY_ENDPOINT=<relay-hostname>:<relay-port>
+RELAY_TLS=false
+RELAY_TLSCA_CERT_PATH=<path_to_tls_ca_cert_pem_for_relay>
 NETWORK_NAME=<network-name>
 DRIVER_PORT=<driver-server-port>
+DRIVER_TLS=false
+DRIVER_TLS_CERT_PATH=<path_to_tls_cert_pem_for_driver>
+DRIVER_TLS_KEY_PATH=<path_to_tls_key_pem_for_driver>
```

Comment:
Add these environment variables in `docker-compose.yml` as well. Something like this:
```
environment:
  - RELAY_TLS=${RELAY_TLS}
  - RELAY_TLSCA_CERT_PATH=${RELAY_TLSCA_CERT_PATH}
```

Reply:
Yes, I need to do this. And test end-to-end with containerized relays and drivers, which I have not done yet.